docker-run-opencv-ovms.sh

```diff
@@ -5,6 +5,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+xhost +local:docker
```

Comment:
We should add this to the RENDER_MODE check in line 125. Setting this by default could cause some security concerns

Reply:
done